ports/espressif/boards/bpi_leaf_s3/mpconfigboard.mk

```diff
@@ -5,9 +5,13 @@ USB_MANUFACTURER = "BananaPi"
 
 IDF_TARGET = esp32s3
 
+CIRCUITPY_ESP_FLASH_SIZE=8MB
 CIRCUITPY_ESP_FLASH_MODE=dio
 CIRCUITPY_ESP_FLASH_FREQ=80m
-CIRCUITPY_ESP_FLASH_SIZE=8MB
+
+CIRCUITPY_ESP_PSRAM_SIZE = 2MB
+CIRCUITPY_ESP_PSRAM_MODE = qio
+CIRCUITPY_ESP_PSRAM_FREQ = 80m
```

Comment:
2mb @ 80m? I would request this is rechecked.

Reply:
This was the old setting in sdkconfig: https://github.com/adafruit/circuitpython/pull/8411/files/c86b3ae19f58dc1420f6924272a24cb380eb6021#diff-3ceb32b0ad9d81881892c47ecb62843a3350402331202b1113d1769cd82b2294L28

Reply:
I know, but it's particularly sus, considering no other boards other than the bpi's have such settings.
if it's a `r2` chip as most 2mb things are, the settings are wrong.
I just felt this needed to be noticed. It could just be a weird, one off chip.